polars/src/lib.rs

```diff
@@ -198,6 +198,8 @@
 #![allow(dead_code)]
 #![feature(iterator_fold_self)]
 #![feature(doc_cfg)]
+#![feature(box_patterns)]
```

Comment:
Could you make it work without a new unstable feature? Arrow is being stabilized and I want to go to stable as soon as Arrow is.

Reply:
Yes